src/sections/homepage/search-input/SearchInput.tsx

```diff
@@ -0,0 +1,68 @@
+import { useRef, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Form } from '@iqss/dataverse-design-system'
+import { CloseButton } from 'react-bootstrap'
+import { Search as SearchIcon } from 'react-bootstrap-icons'
+import { QueryParamKey, Route } from '../../Route.enum'
+import styles from './SearchInput.module.scss'
+
+export const SearchInput = () => {
+  const navigate = useNavigate()
+  const inputSearchRef = useRef<HTMLInputElement>(null)
+  const [searchValue, setSearchValue] = useState('')
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(event.target.value)
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    const trimmedSearchValue = searchValue.trim()
+
+    if (!trimmedSearchValue) return
+
+    const encodedSearchValue = encodeURIComponent(trimmedSearchValue)
+
+    const searchParams = new URLSearchParams()
+    searchParams.set(QueryParamKey.QUERY, encodedSearchValue)
+
+    const collectionUrlWithQuery = `${Route.COLLECTIONS_BASE}?${searchParams.toString()}`
+
+    navigate(collectionUrlWithQuery)
+  }
+
+  const handleClearSearch = () => {
+    setSearchValue('')
+    inputSearchRef.current?.focus()
+  }
+
+  return (
+    <Form onSubmit={handleSubmit} className={styles['search-input-wrapper']}>
+      <div className={styles['input-and-clear-wrapper']}>
+        <Form.Group.Input
+          type="text"
+          role="search"
+          aria-label="Search"
+          autoFocus
+          autoComplete="off"
+          className={styles['text-input']}
+          value={searchValue}
+          onChange={handleSearchChange}
+          ref={inputSearchRef}
+        />
+        {searchValue && (
+          <CloseButton
```

Comment:
Can we use the design system button here rather than a react-bootstrap component?